From 6e398efbe22fb5ca84cda66b27337e44c918c9fb Mon Sep 17 00:00:00 2001
From: jzhuan5 <jin.can.zhuang@intel.com>
Date: Thu, 24 Jan 2013 13:00:55 -0500
Subject: [PORT FROM MAIN] usb:adbd: Don't compile exit_on_close for adb host

BZ: 82358

exit_on_close flag is for adbd only, and shouldn't be compiled when
compiling adb host.

Change-Id: I6305a56adf832c3a6446f9d241930f1e67009ba3
Orig-Change-Id: I3e1e04375accac5566261b7d66c64801645927ec
Signed-off-by: jzhuan5 <jin.can.zhuang@intel.com>
Reviewed-on: http://android.intel.com:8080/88594
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Tang, Jianqiang <jianqiang.tang@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Sun, Jianhua <jianhua.sun@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 adb/sockets.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/adb/sockets.c b/adb/sockets.c
index 587bfa4..f0ee457 100644
--- a/adb/sockets.c
+++ b/adb/sockets.c
@@ -199,7 +199,9 @@ static void local_socket_close(asocket *s)
 static void local_socket_destroy(asocket  *s)
 {
     apacket *p, *n;
+#if !ADB_HOST
     int exit_on_close = s->exit_on_close;
+#endif
 
     D("LS(%d): destroying fde.fd=%d\n", s->id, s->fde.fd);
 
@@ -217,11 +219,13 @@ static void local_socket_destroy(asocket  *s)
     remove_socket(s);
     free(s);
 
+#if !ADB_HOST
     if (exit_on_close) {
         D("local_socket_destroy: exiting\n");
         sleep(1);
         exit(1);
     }
+#endif
 }
 
 
-- 
1.7.4.1

